packages/context/src/__tests__/unit/binding-inspector.unit.ts

```diff
@@ -154,6 +155,72 @@ describe('createBindingFromClass()', () => {
     expect(ctx.getSync(binding.key)).to.eql('my-value');
   });
 
+  it('inspects dynamic value provider classes', () => {
+    const spec = {
+      tags: ['rest'],
+      scope: BindingScope.CONTEXT,
+    };
+
+    @bind(spec)
```

Comment:
Please add some documentation for LB4 app and extension developers to explain how to use `@bind` to tweak how is a dynamic value provider class bound to the context.

Reply:
Added docs.